### DISCUSSION TITLE
Adding a glow effect to the window title text

### DISCUSSION THREAD

**Discussion by Umbrella-TEC** [Maplespe/DWMBlurGlass#172] [Ideas]
Please consider adding a glow effect to the window title text.

Windows itself determines whether the text will be white or black. When transparency is turned on, the text is black and difficult to see if the wallpaper is dark.

In version 22h2, glow no longer works in theme styles.

Screenshot from version 1809:
![2024-02-15 15_42_11-10  Работает  - Oracle VM VirtualBox](https://github.com/Maplespe/DWMBlurGlass/assets/159888040/03d5fdca-eb6a-4c33-ab44-6d14401d0f43)

Screenshot from version 22h2:
![2024-02-15 15_01_07-1022  Работает  - Oracle VM VirtualBox](https://github.com/Maplespe/DWMBlurGlass/assets/159888040/60551140-5601-471e-b5b8-1c0baa4f2493)

**Comment by Maplespe** [comment]
Which 22H2 are you talking about?
If it is Windows 10 22H2 it is supported.
If it's Windows 11 22H2 it's not supported.
But I talked about those in #75  and #86 .
If you want to use Windows 11 the last one that supports text glow is 21H2.

**Comment by Umbrella-TEC** [comment]
Windows 10 22H2: text glow doesn't work even though it was added to the theme style

**Comment by Maplespe** [comment]
This should be a problem with your theme, I use the [Aero10 theme](https://www.deviantart.com/vaporvance/art/Aero10-for-Windows-10-1903-22H2-909711949).
It works all right.
![194918](https://github.com/Maplespe/DWMBlurGlass/assets/43676879/c590ea6e-164b-4f32-8ea8-ad603242c15a)

**Comment by Umbrella-TEC** [comment]
Thank you! Aero10 is very well made, I will use it
